src/hooks/use-is-valid-blueprint.ts

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState, useMemo } from 'react';
+import { getIpcApi } from 'src/lib/get-ipc-api';
+import type { Blueprint } from 'src/stores/wpcom-api';
+
+export function useIsValidBlueprint( className?: string, content?: string ) {
+	const blueprintJson = useMemo< Blueprint[ 'blueprint' ] | null >( () => {
+		if ( ! className?.includes( 'language-json' ) ) {
+			return null;
+		}
+		if ( ! content ) {
+			return null;
+		}
```

Comment:
```suggestion
		if ( ! className?.includes( 'language-json' ) || ! content ) {
			return null;
		}
```
We can simplify a logic by the moving condition.

Reply:
Thanks, I applied the suggestion. 👍